backend/src/main/java/de/bund/digitalservice/ris/search/utils/HttpLogSanitizer.java

```diff
@@ -1,21 +1,57 @@
 package de.bund.digitalservice.ris.search.utils;
 
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URLDecoder;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
 import org.json.JSONObject;
+import org.zalando.logbook.HttpRequest;
 
 public class HttpLogSanitizer {
 
   private HttpLogSanitizer() {
     throw new IllegalStateException("Utility class");
   }
 
-  public static JSONObject sanitizeLogJson(JSONObject json) {
+  public static JSONObject sanitizeLogJson(JSONObject json, HttpRequest request) {
```

Comment:
I think we should leave this method as-is and move `json.put("queryParams", ...)` to `LogRequestAndResponseTogetherSink::write` to keep the separation of concerns neat.